openshift/4.0/templates/jenkins/config.yaml

```diff
@@ -0,0 +1,114 @@
+kind: DeploymentConfig
```

Comment:
Added for ease of updating Jenkins resources so that it can build/deploy without timing out.